doc/source/setup-helm.rst

```diff
@@ -26,7 +26,13 @@ Initialization
 --------------
 
 After installing helm on your machine, initialize helm on your Kubernetes
-cluster. At the terminal, enter:
+cluster. There are two pathways to do this. One if you are running on Microsoft
+Azure, and another if you're running on another provider.
+
+If you are not on Azure
```

Comment:
hmm, I'm loathe to make this happen for several reasons:

1. It talks about particular vendors rather than particular features deep inside the docs
2. It is only true for AKS, not all of Azure (there are other ways to set up Kubernetes this on Azure).

I think I'm happy mentioning Azure AKS as an example here, but not as a subheading.

Reply:
Sounds good - mostly I did this because the current instructions are confusing as to what we mean by "step 2". Is it the second section of the `setup-helm` page? Or is it *just* number "2" within the "initialization" section https://zero-to-jupyterhub-with-kubernetes.readthedocs.io/en/latest/setup-helm.html#initialization

Reply:
it's only number 2 within the initialization section.

I agree it's confusing, but given AKS is in preview now and they're constantly fixing it I think it's better to keep our vendor specific stuff confined...

Reply:
You *could* skip the whole thing and just run `helm init` actually, 'just skip step 2' was the 'minimal diff from what we do for more mature vendors'.

Reply:
I'm happy to keep the instructions the same, just think they could be more explicit. I'll push that change in a sec